common/src/main/res/values-ca/strings.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright 2022 Dash Core Group.
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<resources>
+    <string name="button_retry">Re-intenta</string>
+</resources>
```

Comment:
I am curious where these translations came from.  `ca` is a language we have nothing for in `common` currently.

Reply:
Nevermind, these strings were moved to `common` from other places.

In Transifex, when we push the update via the source (removing text from one file, adding it to another in english), then in the translations the "new" text in `common` will be auto translated from what was in "memory" from before.